### DISCUSSION TITLE
Proxying path not working since Nova 4

### DISCUSSION THREAD

**Discussion by baszach** [laravel/nova-issues#4658] [Q&A] [answered]
Hello together,

I have the following middleware in my Nova project. It checks if the `request` path matches the pattern and shows a documentation page. Since Nova 4 it doesn't work anymore and the `request` is empty when I display it with `dd(...)` or `info(...)`. I don't believe this is a bug but I don't understand what's going on behind the curtains.

How can I rebuild this function with Nova's new routing? Any help would be really appreciated! Thanks! 

>     public function handle($request, Closure $next, $guard = null)
>     {
>         if ($request->is('docs*')) {
>             if (Auth::check()) {
>                 $proxiedPath = Str::of(preg_replace(
>                     '/\/docs\/?/',
>                     '',
>                     $request->getPathInfo(),
>                 ))->whenEmpty(fn ($str) => 'index.html');
>                 return Router::toResponse($request, Http::get(config('app.docs_host').'/'.$proxiedPath));
>             }
>         }
>         return $next($request);
>     }

**Comment by crynobone** [comment, marked as answer]
Laravel Nova 4 doesn't have a catch all route, so you just need to register `docs` route manually in `routes/web.php`.

**Reply by baszach** [reply]
Great thanks.
